tools/reviewer/job/sync/github_api.proto

```diff
@@ -0,0 +1,136 @@
+syntax = "proto3";
+
+package com.google.startupos.tools.reviewer.job.sync;
+
+option java_package = "com.google.startupos.tools.reviewer.job.sync";
+option java_outer_classname = "GitHubProtos";
+
+// GitHub User
+message GHUser {
+  int64 id = 1;
+  string email = 2;
+  string name = 3;
+  string login = 4;
+}
+
+message GHUserReq { string login = 1; }
```

Comment:
Let's remove the `GH` prefix (the import has `GitHubProtos` already).
Also, let's s/Req/Request, s/Resp/Response